jenkinsfiles/grant_access_to_app.groovy

```diff
@@ -1,28 +1,41 @@
 node {
 
+    properties([
+        parameters([
+            string(
+                name: 'APP_NAME',
+                description: 'Shiny App name',
+                defaultValue: ''),
+            string(
+                name: 'EMAILS',
+                description: 'Semicolon delimited list of email addresses',
+                defaultValue: ''),
+        ])
+    ])
+
     stage ("Git checkout") {
         checkout scm
     }
 
     // Creates passwordless user in Auth0 and adds to app group
     // NOTE: Groups provided by Auth0 Authorization Extension
     stage ("Create Auth0 passwordless user and add to app group") {
+      env.AUTHZ_API_ID = 'urn:auth0-authz-api'
       withCredentials([
-        usernamePassword(
-            credentialsId: 'auth0-mgmt-api',
-            usernameVariable: 'CLIENT_ID',
-            passwordVariable: 'CLIENT_SECRET'),
         string(
-            credentialsId: 'authz_api_url',
-            variable: 'AUTHZ_API_URL'),
+            credentialsId: 'AUTH0_CLIENT_ID',
```

Comment:
Shouldn't we convert this to use `usernamePassword` as we do [in the other jenkins job](https://github.com/ministryofjustice/analytics-platform-ops/blob/2a50d01359b328d3f45860c0e5961302fb46c327/jenkinsfiles/grant_access_to_app.groovy#L11)?